### ISSUE TITLE
Mixing computed properties and literal properties

### ISSUE THREAD

**Issue by kentor** [6to5/6to5#378]
Seems like ending an object literal with a computed property causes other properties not to be defined?

```javascript
var obj = {
  first: "first",
  ["second"]: "second",
};
```

generates

```javascript
"use strict";

var _defineProperty = function (obj, key, value) {
  return Object.defineProperty(obj, key, {
    value: value,
    enumerable: true,
    configurable: true,
    writable: true
  });
};

var obj = _defineProperty({}, "second", "second");
```

while

```javascript
var obj = {
  ["second"]: "second",
  first: "first",
};
```

generates

```javascript
"use strict";

var _defineProperty = function (obj, key, value) {
  return Object.defineProperty(obj, key, {
    value: value,
    enumerable: true,
    configurable: true,
    writable: true
  });
};

var obj = (function () {
  var _obj = {};

  _defineProperty(_obj, "second", "second");

  _defineProperty(_obj, "first", "first");

  return _obj;
})();
```

**Comment by timoxley** [comment]
Reproduced in 2.4.8.

**Comment by sebmck** [comment]
Gah, using the wrong array to build properties for the object literal. I'll push out a new version shortly, sorry!

These issues should be biten before they're even released in the future as following the [esnext merger](https://twitter.com/thejameskyle/status/551474226708766720) I've integrated their entire test suite. I'm currently working on integrating the entire Traceur test suite.

**Comment by sebmck** [comment]
Fixed as of 2.4.9
